javascript.js

```diff
@@ -0,0 +1,20 @@
+// set variavles
```

Comment:
The file has increased in 20 lines.
<!-- lookout comment separator -->
---
This line exceeded 13 chars.